install/Makefile

```diff
@@ -1,33 +0,0 @@
-TOP = ..
-include $(TOP)/configure/CONFIG
-
-ioc_name := $(shell basename $(realpath ${TOP}))
-
-SCRIPTS_FOLDER = ../scripts
-IOC_NAME ?= ${ioc_name}
-INSTALL_PREFIX ?= /opt/epics/startup/ioc
-export IOC_NAME
-export INSTALL_PREFIX
-
-install: scripts_install
-	$(TOP)/install/install.sh ${INSTALL_PREFIX}/${IOC_NAME} $(TOP)
-
-uninstall: scripts_uninstall
-	$(TOP)/install/uninstall.sh ${INSTALL_PREFIX}/${IOC_NAME} $(TOP)
-
-scripts_install:
-	$(MAKE) -C ${SCRIPTS_FOLDER} install
-
-scripts_uninstall:
-	$(MAKE) -C ${SCRIPTS_FOLDER} uninstall
```

Comment:
You remove these rules, but do not remove their target folder (i.e. `scripts`).

IMHO, this removal could be done in the same commit that adds the deploy compose file if we decide to follow that approach. This should make clearer that `compose.yml` substitute what systemd services were previously doing (with the except of actually launching the containers).